app/views/workshops/show.html.haml

```diff
@@ -11,19 +11,20 @@
         %small #{humanize_date(@workshop.date_and_time, @workshop.ends_at, with_time: true)} #{@workshop.distance_of_time}
       - if @workshop.date_and_time.past?
         %span.badge.bg-danger= t('messages.already_taken_place')
+
   .row.mt-3
-    .col
+    .col.col-md-9
```

Comment:
Do you need both classes on this line? From memory, I think you only need one `.col-md-9`